extensions/amp-3d-gltf/amp-3d-gltf.md

```diff
@@ -0,0 +1,90 @@
+<!--
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="`amp-3d-gltf`"></a> `amp-3d-gltf`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>Displays GL Transmission Format (gITF) 3D models.</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td>Experimental</td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-3d-gltf" src="https://cdn.ampproject.org/v0/amp-3d-gltf-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>fill, fixed, fixed-height, flex-item, responsive</td>
+  </tr>
+</table>
+
+## Usage
+
+The `amp-3d-gltf` component displays 3D models that are in gITF format.
+
+**Note**: A WebGL capable browser is required to display these models.
+
+###  Example
+
+```html
+<amp-3d-gltf
+    layout="responsive"
+    width="320"
+    height="240"
+    alpha="true"
+    antialiasing="true"
+    src="path/to/model.glb"></amp-3d-gltf>
+```
+
+### Limitations
+
+Currently, only works with glTF 2.0.
+
+Unsupported features:
+- embeded cameras
+- animation
+
+## Attributes
+
+##### src [required]
+A required attribute that specifies the URL to the gltf file.
+
+##### alpha [optional]
+
+A Boolean attribute that specifies whether to see through free space on canvas. By default, free space is filled with black.
```

Comment:
Please confirm my re-wording is correct.  Also, if "see through free space on canvas" really means, the "free space on the canvas is transparent", please rewrite it as so.  Right now the "see through free space on canvas" is hard to parse.

Reply:
You are correct both about your rewording, and about "see through". I changed it as you asked.